src/python/fuzzing/fuzzer_selection.py

```diff
@@ -80,10 +80,13 @@ def get_fuzz_task_payload(platform=None):
     queue_override = environment.get_value('QUEUE_OVERRIDE')
     platform = queue_override if queue_override else environment.platform()
 
-  query = data_types.FuzzerJob.query()
-  query = query.filter(data_types.FuzzerJob.platform == platform)
+  query = data_types.FuzzerJobs.query()
```

Comment:
This can introduce a 30 min delay for a job to run which can be confusing when run locally. I can add a condition here to check for local server to use the individual FuzzerJobs instead, but that can be a bit hacky. Alternatively we can have something a bit smarter which flips some dynamic flag in Config based on how many FuzzerJobs we detect. @mbarbella-chromium @inferno-chromium WDYT?

Reply:
What about local development, maybe we need a bailout that does regular way for environment.is_local. for remote, waiting 30 min is fine.

Reply:
Just added an is_local check.